application/webapp-service/src/main/java/webapp/tier/healthcheck/HealthCheckUtility.java

```diff
@@ -0,0 +1,21 @@
+package webapp.tier.healthcheck;
+
+import java.util.logging.Logger;
+
+import org.eclipse.microprofile.health.HealthCheckResponse;
+
+public class HealthCheckUtility {
```

Comment:
Codacy found an issue: [The utility class name 'HealthCheckUtility' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=5997249)